GetCertificate/__tests__/__snapshots__/printer_recovery_detail_en.test.ts.snap

```diff
@@ -36,3 +36,40 @@ For more information and to download the certificate in printable PDF format go
 <br/><br/><br/><br/><br/>
 "
 `;
+
+exports[`Printer - Recovery - Detail - en_GB should print its markdown with placeholder, if some value is not in valueset 1`] = `
+"
+## Recovery Certificate  
+
+Disease or agent the citizen has recovered from  
+**---**  
```

Comment:
now we got a placeholder "---" instead of raising a decode error and skipping markdown generation